src/animation.jl

```diff
@@ -21,9 +21,9 @@ function frame(anim::Animation, plt::P=current()) where P<:AbstractPlot
     push!(anim.frames, filename)
 end
 
-giffn() = (isijulia() ? "tmp.gif" : tempname()*".gif")
-movfn() = (isijulia() ? "tmp.mov" : tempname()*".mov")
-mp4fn() = (isijulia() ? "tmp.mp4" : tempname()*".mp4")
+giffn() = (isijulia() ? "tmp_"*randstring()*".gif" : tempname()*".gif")
```

Comment:
Have you tried 
```julia
giffn() = tempname() * ".gif"
```
instead as well? Is it necessary to have a different behavior for IJulia?

Reply:
Yes, I did. That made nothing show up inline at all. I'm not sure why nothing showed up, but I suspected that there was some reason for that and that is the reason there was a `isijulia()` if there to begin with. I tested it both with `gif()` and `mp4()`, both showed nothing with `tempname()`

If that is the preferred solution I can try and figure out why, and see if I can work around it.

Reply:
I was just curious. Whatever works is fine for me.